docs/pages/radiogroup.js

```diff
@@ -0,0 +1,986 @@
+      />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Direction"
+          description="RadioGroups can be shown in a column or row by specifying the `direction` property."
+        >
+          <MainSection.Card
+            defaultCode={`
+function RadioButtonExample() {
+  const [favorite, setFavorite] = React.useState();
+  const [favoriteFood, setFavoriteFood] = React.useState();
+
+  return (
+    <Flex gap={8}>
+      <RadioGroup legend="What is your favorite pet?" id="directionExample-1">
+        <RadioGroup.RadioButton
+          checked={favorite === 'dogs'}
+          id="favoriteDog"
+          label="Dogs"
+          name="favorite"
+          onChange={() => setFavorite('dogs')}
+          value="dogs"
+        />
+        <RadioGroup.RadioButton
+          checked={favorite === 'cats'}
+          id="favoriteCat"
+          label="Cats"
+          name="favorite"
+          onChange={() => setFavorite('cats')}
+          value="cats"
+        />
+        <RadioGroup.RadioButton
+          checked={favorite === 'plants'}
+          id="favoritePlants"
+          label="Plants"
+          name="favorite"
+          onChange={() => setFavorite('plants')}
+          value="plants"
+        />
+      </RadioGroup>
+
+      <RadioGroup legend="What is your favorite snack?" errorMessage="Please select one" direction="row" id="directionExample">
+        <RadioGroup.RadioButton
+          checked={favoriteFood === 'pizza'}
+          id="favoritePizza"
+          label="Pizza"
+          name="favoriteFood"
+          onChange={() => setFavoriteFood('pizza')}
+          value="pizza"
+        />
+        <RadioGroup.RadioButton
+          checked={favoriteFood === 'curry'}
+          id="favoriteCurry"
+          label="Curry"
+          name="favoriteFood"
+          onChange={() => setFavoriteFood('curry')}
+          value="curry"
+        />
+        <RadioGroup.RadioButton
+          checked={favoriteFood === 'sushi'}
+          id="favoriteSushi"
+          label="Sushi"
+          name="favoriteFood"
+          onChange={() => setFavoriteFood('sushi')}
+          value="sushi"
+        />
+      </RadioGroup>
+    </Flex>
+  );
+}
+          `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Size"
+          description="RadioButtons can be either `sm` (16px) or `md` (24px), which is the default."
+        >
+          <MainSection.Card
+            defaultCode={`
+function RadioButtonExample() {
+  const [favorite, setFavorite] = React.useState();
+  const [favoriteFood, setFavoriteFood] = React.useState();
+
+  return (
+    <Flex gap={8}>
+      <RadioGroup legend="What is your favorite snack?" errorMessage="Please select one" id="sizeExample">
+        <RadioGroup.RadioButton
```

Comment:
should we use the recommended one? `To continue you must select one item from this list`

Reply:
same here